system/web/Renderer.cfc

```diff
@@ -694,6 +694,29 @@ component accessors="true" serializable="false" extends="coldbox.system.Framewor
 		return locateView(arguments.view);
 	}
 
+	public string function elixir( required string fileName, string buildDirectory = 'build' ) {
+		var includesConvention = controller.getSetting( "includesConvention", true );
```

Comment:
You will need to also make this smarter for modules.  So basically, you need to check if you are in a module call or not.

You can use the `event.getCurrentModule()` and if that is empty, that means you go to the parent app, else you can go into the module using: `event.getModuleRoot()` + the convention `includes`.